models/object_detection_nanodet/demo.py

```diff
@@ -0,0 +1,110 @@
+import cv2
+import numpy as np
+import argparse
+import time
+import matplotlib.pyplot as plt
+from NanodetPlus import NanoDet
+
+backends = [cv2.dnn.DNN_BACKEND_OPENCV, cv2.dnn.DNN_BACKEND_CUDA]
+targets = [cv2.dnn.DNN_TARGET_CPU, cv2.dnn.DNN_TARGET_CUDA, cv2.dnn.DNN_TARGET_CUDA_FP16]
+help_msg_backends = "Choose one of the computation backends: {:d}: OpenCV implementation (default); {:d}: CUDA"
+help_msg_targets = "Chose one of the target computation devices: {:d}: CPU (default); {:d}: CUDA; {:d}: CUDA fp16"
+
+try:
+    backends += [cv2.dnn.DNN_BACKEND_TIMVX]
+    targets += [cv2.dnn.DNN_TARGET_NPU]
+    help_msg_backends += "; {:d}: TIMVX"
+    help_msg_targets += "; {:d}: NPU"
+except:
+    print('This version of OpenCV does not support TIM-VX and NPU. Visit https://gist.github.com/Sidd1609/5bb321c8733110ed613ec120c7c02e41 for more information.')
+
+with open('coco.names', 'rt') as f:
+    classes = f.read().rstrip('\n').split('\n')
+
+def drawPred(frame, classId, conf, left, top, right, bottom):
+    cv2.rectangle(frame, (left, top), (right, bottom), (0, 0, 0), thickness=2)
+    #label = '%.2f' % conf
+    label =''
+    label = '%s%s' % (classes[classId], label)
+    labelSize, baseLine = cv2.getTextSize(label, cv2.FONT_HERSHEY_SIMPLEX, 0.5, 1)
+    top = max(top, labelSize[1])
+    # cv.rectangle(frame, (left, top - round(1.5 * labelSize[1])), (left + round(1.5 * labelSize[0]), top + baseLine), (255,255,255), cv.FILLED)
+    cv2.putText(frame, label, (left, top - 10), cv2.FONT_HERSHEY_SIMPLEX, 1, (0, 255, 0), thickness=2)
+    return frame
+
+if __name__=='__main__':
+    parser = argparse.ArgumentParser(description='Nanodet inference using OpenCV an contribution by Sri Siddarth Chakaravarthy part of GSOC_2022')
```

Comment:
Make the model path as one of the argument as shown below:
https://github.com/opencv/opencv_zoo/blob/54190541bb9f717144b255358ca6b37e3e47bf23/models/face_detection_yunet/demo.py#L36

Reply:
included modelpath as args "model"